pkg/controller/migmigration/task.go

```diff
@@ -1023,7 +1023,8 @@ func (t *Task) Run(ctx context.Context) error {
 	case Canceling:
 		// Skip directly to Completed if the Cancel was set on a Rollback migration.
```

Comment:
If I'm understanding the change correctly, after this the migration will be at Canceled rather than Completed -- From EnsureAnnotationsDeleted to Canceled when `next()` is called just below. Maybe the above comment should be changed to reflect this as well.